Accessories/ExpansionBoard_uInverter/REV20210506A/README.md

```diff
@@ -0,0 +1,11 @@
+# uInverter Board
```

Comment:
This file should include a table of the PCB properties for ordering the revision. See the other hardware repos for examples. Examples: board size, layers, min clearance, etc.

Reply:
Included the table of PCB relevant details for PCB ordering.